servers/partner-ci/builders/workstation.sh

```diff
@@ -94,26 +94,63 @@ print data
 client.close()
 CONTENT
 
-chmod +x $SSH_ENDPOINT
+sudo chmod +x $SSH_ENDPOINT
 
 }
 
 configure_nfs(){
   set -x
   create_ssh_endpoint
-  source $VENV_PATH/bin/activate
   for esxi_host in $ESXI_HOSTS; do
-
-    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'storages=$(esxcli storage nfs list | grep nfs | cut -d" " -f1); for storage in $storages; do esxcli storage nfs remove -v $storage; done'
-    echo "nfs storages have been successfully removed for $esxi_host"
+    echo "restart services to avoid connection problems>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD './sbin/services.sh restart'
+    echo "restart scfsbd-watchdog>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD '/etc/init.d/sfcbd-watchdog stop; rm -rf /var/run/sfcb/*;/etc/init.d/sfcbd-watchdog start'
+    echo "esxcli network firewall set --enabled false, to avoid connection problems>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli network firewall set --enabled false'
+    echo "system syslog hostname get>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli system hostname get'
+    echo "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+    if [[ ${DEBUG} == "true" ]]; then
+      echo "system syslog config get>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+      python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli system syslog config get'
```

Comment:
refactor to function

remote_exec (){
host=$1
command=$2
 python2 $SSH_ENDPOINT $host $ESXI_USER $ESXI_PASSWORD $cmd
 echo "$cmd-------------------------------------------------------"
}